migrations/sql/V2017.14.11.1716__AddModelRunParameters.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE upload_info(
+id SERIAL,
+uploaded_by TEXT NOT NULL,
+uploaded_on TEXT NOT NULL
+)
+
+CREATE TABLE model_run_parameter_set(
+   id SERIAL,
+   responsibility_set INTEGER NOT NULL,
+   description TEXT,
+   upload_info INTEGER NOT NULL,
+   model_version INTEGER NOT NULL
+);
+
+ALTER TABLE model_run_parameter_set ADD FOREIGN KEY (responsibility_set) REFERENCES responsibility_set.id;
+ALTER TABLE model_run_parameter_set ADD FOREIGN KEY (model_version) REFERENCES model_version.id;
+ALTER TABLE model_run_parameter_set ADD FOREIGN KEY (upload_info) REFERENCES upload_info.id;
+
+CREATE TABLE model_run(
+intenal_id SERIAL,
+run_id TEXT NOT NULL,
+model_run_parameter_set INTEGER NOT NULL
```

Comment:
Should be a serial, I think

Reply:
this one's a foreign key to another table, we don't need/want it to autoincrement?

Reply:
You're right. Sorry; I thought `serial` was more than just an auto-incrementing integer.